snap/README.md

```diff
@@ -0,0 +1,39 @@
+This directory contains all of the files required to build a ZAP 2.7.0 https://snapcraft.io/ snap.
+
+Docker is used to build the snap as snapcraft is only supported on a limited number of linux distros.
+
+In order to build the snap run:
+
+`./build-snap.sh`
```

Comment:
We could use `docker run -it -v "$PWD:$PWD" -w "$PWD" snapcore/snapcraft snapcraft` directly, which builds the snap in the local file system (in this case we should add the snap dirs and the snap file to `.gitignore`).